tests/test_core/test_xy.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
```

Comment:
Do you mean that this order is alright and should be continued in the other test files?

Reply:
The empty test files were meant to illustrate the testing skeleton. Should I remove them anyway for this pull request?

Reply:
Separating the contest.py for each submodule can be useful to have a better overview over the fixtures. But if both submodules share a lot features it could make sense to merge them into one conftest.py file, which lays one layer higher in the folder structure.

Reply:
@draabe see commits below